### PR TITLE
[WAZO-1291] Integrate Async API

### PR DESCRIPTION
**Pull request by bogue**

### CODE REVIEW

File: src/component/documentation/async-api.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+};
+
+const getServiceName = (raw) => {
+  const url = new URL(raw);
+  const path = url.pathname.split('/');
+  return path[2];
+}
+
+const Page = ({ pageContext: { moduleName, module, modules, auth_url } }) => (
+  <Layout pageTitle={`Console - ${module.title}`} breadcrumbs={[{ link: '/install', label: 'Install', active: true }]} className="body-green">
```
Comment (chartrandf) [inline src/component/documentation/async-api.js:31]:
```suggestion
  <Layout pageTitle={`API Events - ${module.title}`} breadcrumbs={[{ link: '/install', label: 'Install', active: true }]} className="body-green">
```

File: src/component/documentation/async-api.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      <div className="container" style={styles.container}>
+        <div className="list-group" style={styles.listGroup}>
+          {Object.keys(modules).map(m => {
+
+            return modules[m].apiEvents && <Link key={m} to={`/documentation/events/${m}`} className={`list-group-item list-group-item-action ${m === moduleName ? 'disabled' : ''}`}>
+              {modules[m].title}
+              <div style={styles.subtitle}>{modules[m].repository}</div>
+            </Link>
+          })}
+        </div>
```
Comment (chartrandf) [inline src/component/documentation/async-api.js:42]: Open suggestion, to be like API Reference. 
- Remove sub navigation
- Enable sidebar from async (could help to navigate quickly thought all events)

**Review by chartrandf — COMMENTED** [review]
Nice, great addition 🎉  Small adjustments to discuss, after that it's ready to merge

File: src/component/documentation/async-api.js (hunk 2 above)
Comment (chartrandf) [reply src/component/documentation/async-api.js:42]: Disucssed. We will improve this point from in another work.

**Review by chartrandf — APPROVED** [review]
Good job

**Comment by chartrandf** [comment]
recheck

**Comment by bogue** [comment]
recheck

**Comment by sboily** [comment]
We'll see on the new infra, and we need to increase the RAM, we introduced new dependencies for asyncAPI.

**Pull request merged**